src/libs/sfdk/dockercontainer.h

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
```

Comment:
The content of this file should be moved to the *_p.h file - concrete VirtualMachine implementations are not public.

Reply:
fixed